packages/calcite-components-react/package.json

```diff
@@ -6,12 +6,10 @@
   "description": "A set of React components that wrap calcite components",
   "license": "SEE LICENSE.md",
   "scripts": {
-    "build": "rimraf dist && npm run compile",
-    "prebuild": "npm run patch:ssr",
-    "clean": "rimraf dist node_modules .turbo",
-    "compile": "npm run tsc",
+    "build": "tsc",
+    "clean": "rimraf dist",
     "patch:ssr": "ts-node support/patchSSR.ts",
-    "tsc": "tsc"
```

Comment:
Can you undo this change? I'm not sure why Stencil created an npm script for `tsc` in their example, but I'd like to do some tests before removing it, since it's not related to this PR.

https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/example-project/component-library-react/package.json#L17

Let's keep unrelated changes in separate PRs, especially when they are for different packages in the monorepo